lib/gui/components/update-notifier/styles/_update-notifier.scss

```diff
@@ -20,3 +20,7 @@
   margin-bottom: 0;
 }
 
+.modal-update-notifier .modal-body a {
```

Comment:
Oh, so we are not using the pseudo classes mentioned above at all here in Etcher (they have all been normalized to the "normal state"), so just ignore me on this one :)